Schema_refinery/schema_refinery.py

```diff
@@ -0,0 +1,125 @@
+"""
+Purpose
+-------
+This file call all other modules of the packages, containing several arguments
+as input depending on the desired module.
+
+Code documentation
+------------------
+"""
+import sys
+import argparse
+
+try:
+    from DownloadAssemblies import download_assemblies
+except ModuleNotFoundError:
+    from Schema_refinery.DownloadAssemblies import download_assemblies
+
+def download_module():
+    """
+    Function that contains the required arguments for the download_module of the
+    schema refinery package, this function exports the arguments to
+    download_asseblies.py
+    """
+    parser = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('Download_module', nargs='+',
+                        help='')
+
+    #Common arguments between databases
+    parser.add_argument('-db', '--database', type=str,
+                        required=True, dest='database',
+                        nargs='+',
+                        choices = ['NCBI','ENA661K'],
+                        help='Databases from which assemblies will be downloaded. ')
+
+    parser.add_argument('-s', '--species', type=str,
+                        required=True, dest='species',
+                        help='Desired species to fetch')
+
+    parser.add_argument('-o', '--output-directory', type=str,
+                        required=True, dest='output_directory',
+                        help='Output Path')
+
+    parser.add_argument('-th', '--threads', type=int,
+                        required=False, default=2,
+                        dest='threads',
+                        help='Number of threads for download.')
+
+    parser.add_argument('-r', '--retry', type=int,
+                        required=False, dest='retry',
+                        default=7,
+                        help='Maximum number of retries when a '
+                             'download fails.')
+
+    parser.add_argument('-e', '--email', type=str,
+                        required=True, dest='email',
+                        help='email for entrez NCBI')
+
+    parser.add_argument('-k', '--api_key', type=str,
+                        required=False, dest='api_key',
+                        help='API key to increase the mumber of requests')
+
+    parser.add_argument('-fm', '--f_metadata',
```

Comment:
Suggest using `-` between words in the extended format name attributed to parameter names. The long format should also include complete words (e.g.: `--fetch-metadata`).